src/mono/mono/mini/mini-llvm.c

```diff
@@ -303,6 +310,142 @@ static void create_aot_info_var (MonoLLVMModule *module);
 static void set_invariant_load_flag (LLVMValueRef v);
 static void set_nonnull_load_flag (LLVMValueRef v);
 
+enum {
+	INTRIN_scalar = 1 << 0,
```

Comment:
Is there any particular reason we are defining some of these with constant bit shifts, some with decimal literals, and some with hex literals?